File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/CLSYeuCau_PACSService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CLSYeuCau_PACSService : BaseService<CLSYeuCau_PACS, ICLSYeuCau_PACSRepository>
    , ICLSYeuCau_PACSService
    {
    private readonly ICLSYeuCau_PACSRepository _CLSYeuCau_PACSRepository;
    public CLSYeuCau_PACSService(ICLSYeuCau_PACSRepository CLSYeuCau_PACSRepository) : base(CLSYeuCau_PACSRepository)
    {
    _CLSYeuCau_PACSRepository = CLSYeuCau_PACSRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/CLSYeuCau_PACS.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CLSYeuCau_PACS } from './CLSYeuCau_PACS.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CLSYeuCau_PACSService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'PACS_ID', 'CLSYeuCau_ID'];

    List: CLSYeuCau_PACS[] | undefined;
    ListFilter: CLSYeuCau_PACS[] | undefined;
    FormData!: CLSYeuCau_PACS;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CLSYeuCau_PACS";
    }
}

